social-network-api/src/main/java/fs/socialnetworkapi/dto/login/LoginDtoIn.java

```diff
@@ -0,0 +1,13 @@
+package fs.socialnetworkapi.dto.login;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Data;
+
+@Data
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
```

Comment:
do you need a snake case strategy here? There is no variables with more than one word.